var/spack/packages/spot/package.py

```diff
@@ -0,0 +1,18 @@
+from spack import *
+import os
+
+class Spot(Package):
+    """Spot is a C++11 library for omega-automata manipulation and model checking."""
+    homepage = "https://spot.lrde.epita.fr/index.html"
+    url      = "http://www.lrde.epita.fr/dload/spot/spot-1.99.3.tar.gz"
+
+    version('1.99.3', 'd53adcb2d0fe7c69f45d4e595a58254e')
+
+    #depends_on("gcc@4.8:")
```

Comment:
This is not currently possible, but it only requires the addition of some syntax.  We've debated this on the mailing list a bit -- you can do this for a dependency with:
```
depends_on('foo%gcc@4.8:')
```
For your _self_, what syntax would be good for you?  Would this work?
```
require('%gcc@4.8:')
```